pkgs/tools/networking/wuzz/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, pkgconfig, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "wuzz";
+  version = "89bebba7a4a00c52e97d411a3e82660189adc4ff";
```

Comment:
Git revisions are invalid versions; please use the commit date in ISO 8601 format.